### DISCUSSION TITLE
Welcome to Iterated Dynamics Discussions!

### DISCUSSION THREAD

**Discussion by LegalizeAdulthood** [LegalizeAdulthood/iterated-dynamics#163] [Announcements]
# Welcome!
In the Discussions section of this repository you'll find the developers kibbitzing about implementation plans as well as what other open source fractal renderer projects we plan on stealing code from `:)`.  We welcome discussion from everyone, users and developers alike.

Feel free to start a discussion on any topic relevant to Iterated Dynamics:
- Is there something you don't understand about how Iterated Dynamics works?
- Did you run into a problem?  (Maybe you're not sure if you should file an [issue](https://github.com/LegalizeAdulthood/iterated-dynamics/issues)?)
- Got an idea for a feature you'd like to see?
- Got a suggestion for something you'd like to see incorporated from another open source fractal renderer?
- Something else on your mind, perhaps? `:)`

These topics and more are all welcome in our discussion area.

You will need an account on github to participate in discussions; [joining github](https://github.com/join) is free.

**Comment by marcusrezende** [comment]
In ManpWIN, Paul and I developed an alternative approach to understanding dynamic systems and fractals, based on the assumption that their coordinate systems were incorrectly expressed in cylindrical, spherical, elliptical, and other coordinate systems. By performing these coordinate transformations, we created previously unknown types of fractals and dynamic systems. We referred to them as mathematical fractals and dynamic systems (D.S.). I can send some examples in Basic, the only language I can program.

**Reply by LegalizeAdulthood** [reply]
> [...] based on the assumption that their coordinate systems were incorrectly expressed in cylindrical, spherical, elliptical, and other coordinate systems.

I'm afraid you lost me here.  Why do you say that they were **incorrectly** expressed?  Maybe it would help if you gave a specific example.

Certainly arbitrary coordinate system mappings could be applied at various stages of the pipeline.  The inversion support in Id is an example of this that has been hard-coded, probably because circle inversion is something that is discussed in _Beauty of Fractals_ and other early fractal books.

I think UltraFractal achieves the same effect with a different mechanism.

**Comment by marcusrezende** [comment]
This story began when I plotted an oscillator using the standard Cartesian
coordinate system, only to find that the attractors did not match those
presented in the paper. Upon closer examination, I realized that the
oscillator had been described in a spherical coordinate system. Once I
executed it correctly, the images aligned perfectly with those in the paper.

What followed was an interesting discovery: I observed—later confirmed by
Paul with scape-time fractals—that oscillators could generate different
attractors from the same equations and constants, depending on how the
coordinates were interpreted. Specifically, when 'X' was treated as Rho,
'Y' as Theta, and 'Z' as Phi, and after completing the entire processing
loop, we converted them back to Cartesian coordinates and plotted the new
oscillator.
